qiskit/pulse/schedule.py

```diff
@@ -650,8 +651,10 @@ def assign_parameters(self,
         Returns:
             Schedule with updated parameters (a new one if not inplace, otherwise self).
         """
+        # unroll the parameter dictionary (needed if e.g. it contains a ParameterVector)
+        unrolled_param_dict = self._unroll_param_dict(value_dict)
```

Comment:
I think I prefer this: 
```suggestion
        value_dict = self._unroll_param_dict(value_dict)
```
is there a reason to make it distinct?